client/src/components/geneExpression/gene/index.tsx

```diff
@@ -17,6 +16,7 @@ import { EVENTS } from "../../../analytics/events";
 import { ActiveTab } from "../../../common/types/entities";
 import { State, Props, mapStateToProps, mapDispatchToProps } from "./types";
 import { MINI_HISTOGRAM_WIDTH } from "../constants";
+import { InfoButton, InfoButtonWrapper } from "../../categorical/value/style";
```

Comment:
discussion: wonder if we should create a new folder `client/src/components/common` to host these 💡 ?